cpp/dolfinx/la/SparsityPattern.cpp

```diff
@@ -34,7 +34,8 @@ SparsityPattern::SparsityPattern(
     const std::vector<std::vector<const SparsityPattern*>>& patterns,
     const std::array<std::vector<std::pair<
                          std::reference_wrapper<const common::IndexMap>, int>>,
-                     2>& maps)
+                     2>& maps,
+    const std::array<std::vector<int>, 2>& bs)
```

Comment:
It takes block sizes but sets `_bs({1, 1})` below?

Reply:
The function concatenates sparsity patterns - the input sparsity patterns can have bs > 1, but the  concatenated pattern always has bs = 1. A future optimisation could handle the case where all input sparsity patterns have the same bloc sizes.